helpers/formValidation.tsx

```diff
@@ -17,10 +17,6 @@ const signupValidation = Yup.object({
     .max(TEXT_MAX, `Must be ${TEXT_MAX} characters or less`)
     .trim('Leading and trailing value must be alphanumeric character')
     .required('Required'),
-  password: Yup.string()
```

Comment:
How does this affect the signup password validation?

Reply:
ooooo there's no password inputs for signup anymore :)

Reply:
I meant to move this to a different PR. I don't want to break production on this one.